crates/bevy_ecs/src/event.rs

```diff
@@ -59,6 +59,9 @@ enum State {
 /// [`Events::update`] calls.
 /// Events can be written to using an [`EventWriter`]
 /// and are typically cheaply read using an [`EventReader`].
+/// 
+/// If no ordering is applied between writing and reading systems, there is a risk of a race condition.
+/// This means that whether the events arrive before or after the next [`Events::update`] is unpredictable.
```

Comment:
I would like to see a link to the system ordering docs here if possible.

Reply:
Right, the book seems to include information about race condition. Added

Reply:
I think it makes sense to move this text after the "Each event can be consumed by multiple systems ..." sentence. The current ordering introduces a caveat to parallel execution before even introducing the concept of parallel execution.

Reply:
Done